components/stressPanel.js

```diff
@@ -78,30 +144,118 @@ export default function stressPanel(props) {
-            <IntegerStressParam name="Stress Line Size" description="The non-testing threads will access disjoint memory locations at seperatated by at least this many 32-bit words (values should be between 1 and 128)" paramName="stressLineSize" params={props.params} pageState={props.pageState}/>
-            <IntegerStressParam name="Stress Target Lines" description="How many disjoint memory locations the non-testing threads access in the scratch memory region (values should be between 1 and 128)" paramName="stressTargetLines" params={props.params} pageState={props.pageState}/>
+            {minWorkgroupsDom.jsx}
+            {maxWorkgroupsDom.jsx}
+            {shufflePct.jsx}
+            {barrierPct.jsx}
+            {testMemorySize.jsx}
+            {scratchMemorySize.jsx}
+            {memoryStride.jsx}
+            {memStressPct.jsx}
+            {memStressLoops.jsx}
+            {preStressPct.jsx}
+            {preStressLoops.jsx}
+            {stressLineSize.jsx}
+            {stressTargetLines.jsx}
             <DropdownStressParam name="Stress Assignment Strategy" description="How non-testing threads are assigned to scratch memory regions to access" paramName="stressAssignmentStrategy" params={props.params} options={["round-robin", "chunking"]} updateFunc={stressAssignmentStrategyOnChange} pageState={props.pageState}/>
             <DropdownStressParam name="Memory Stress Pattern" description="The access pattern that non-testing threads access the scratch memory region" paramName="memStressPattern" params={props.params} options={["load-store", "store-load", "load-load", "store-store"]} updateFunc={stressPatternOnChange} pageState={props.pageState}/>
             <DropdownStressParam name="Pre Stress Pattern" description="The access pattern that testing threads access the scratch memory region before executing their litmus test" paramName="preStressPattern" params={props.params} options={["load-store", "store-load", "load-load", "store-store"]} updateFunc={stressPatternOnChange} pageState={props.pageState}/>
           </div>
           <div className="panel-block p-2">
-            <button className="button is-link is-outlined is-fullwidth " style={{ width: "200px", marginLeft: "10px" }}>
-              Reset all Parameters
-            </button>
+            <div className="columns is-2 ">
+              <div className="column ">
+                <div className="buttons are-small">
+                <button className="button is-link is-outlined " onClick={()=>{
+                  console.log("click test1")
+                  click = true;
+                   setParam({...params,minWorkgroups: 4,maxWorkgroups: 4, shufflePct: 0, barrierPct: 0, memStressPct: 0});
+                   props.params.minWorkgroups = 4;
```

Comment:
we should make this a function, and then set the state of each param there. (Same with the other buttons)